components/Themes/src/MaterialThemes.h

```diff
@@ -0,0 +1,18 @@
+/*
+ Copyright 2015-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCTheme.h"
+#import "MDCFeatureHighlightColorThemer.h"
```

Comment:
Doesn't this mean the Theme component will be dependent on every theme-able component? Maybe we should separate the themes and the themers.

theme theme theme theme theme